### ISSUE TITLE
Spacy Entity recognition doesn't match Displacy Entities

### ISSUE THREAD

**Issue by harwee** [explosion/spaCy#1213]
Spacy entity recognition returns different entities compared to displaCy.

Example Spacy:
```
>>> import spacy
>>> nlp = spacy.load("en")
>>> sentence = nlp("Is Google good?")
>>> print(list(sentence.ents))
>>> []
```
On displaCy it correctly recognizes and tags `Google` as  `org`

## Your Environment
* **spaCy version:** 1.9.0
* **Platform:** Linux-4.4.0-1013-aws-x86_64-with-debian-stretch-sid
* **Python version:** 3.6.1
* **Installed models:** en_core_web_md, en

**Comment by Nbharath** [comment]
Did you resolve this problem?

**Comment by harwee** [comment]
Sadly no, looks like displaCy is using a different version of code or language data, than the current available one.

**Comment by ehddnr747** [comment]
Did you resolve this problem? I also have this problem

**Comment by ines** [comment]
Fixed! displaCy now uses the latest v2.0 models: https://demos.explosion.ai/displacy-ent/
